test/test_cpp_extensions_jit.py

```diff
@@ -529,6 +531,38 @@ def compile(code):
         module = compile("int f() { return 789; }")
         self.assertEqual(module.f(), 789)
 
+    @unittest.skipIf(codecs.lookup(locale.getencoding()).name != "utf-8", "Only test in UTF-8 locale")
```

Comment:
I wrote a test for utf-8 locale, but obviously it can only test in os with utf-8 locale and file in gbk encoding. May have a better way.